### DISCUSSION TITLE
How to create a managed configuration for mobile apps on iPads using Jamf School?

### DISCUSSION THREAD

**Discussion by fkm97** [SafeExamBrowser/seb-mac#409] [Q&A]
Hi,

We are attempting to use jamf to create a managed configuration with an SEB config file and am being asked for configuration property override values. Would you happen to know how this is to be done? 

Link for documentation: https://learn.jamf.com/en-US/bundle/jamf-school-documentation/page/Creating_Managed_Configurations_for_Apps.html

**Comment by fkm97** [comment]
is it possible to get any insight for this case?

**Comment by danschlet** [comment]
Sorry, we are very busy and have to prioritize support requests which relate to bugs over such which are user handling questions.

You can do a text search on the [Release Notes](https://safeexambrowser.org/ios/ios_release_notes_en.html) page for "MDM", there are some hints:

- Fixed: Saving config files for MDM didn't use the .plist extension on iOS.
- Added option to export SEB settings as an unencrypted plist file for the MDM Managed Configuration feature (currently supported by iOS SEB clients).
 

1. So you need to make sure to save a SEB configuration for "MDM Managed Configuration" (option available in the current iOS and macOS versions of SEB).
2. Then you can use a text editor to copy the portion between and including the dict tags to the text field in Jamf where you can put the app configuration (as far as I remember it correctly, I know someone ones asked that probably here in the GitHub issues).
`<dict>`
`...`
`</dict>`
3. If configured like that, this SEB configuration will configure the client. This means when users start the SEB app directly, the according MDM distributed config will be used. In case users open a .seb configuration file or a sebs:// link (for example the "Launch Safe Exam Browser" button in Moodle), they will still be using that configuration.

I have no clue why Jamf doesn't seem to have an option to just upload a .plist preferences file (some other MDM systems seem to have that).

Sorry, we also don't have the capacity to improve our manuals or other documentation right now. Usually it's a good tip to search the release notes of the according SEB version or its manuals, sometimes it helps searching the according GitHub repository to find hints how less common features work (especially when third party software like Jamf is involved).

**Comment by acsim** [comment]
Hi,
we are currently also trying to map the configuration via JAMF School. However, when we transfer the plist content, we cannot save the file in JAMF School with the message "Buffer not defined". Even considering the hints given here so far. Does anyone have an idea? 
Many thanks in advance
Peter

**Reply by danschlet** [reply]
I would ask Jamf support. With Jamf Pro the described process worked last time I tried it.

**Comment by acsim** [comment]
Der JAMF Support hat das Anliegen geprüft, der Fehler generiert sich aus folgendem Code-Abschnitt in der exportierten .plist-Datei:

`<key>examKeySalt</key>
<data>
shalfhdsl1490234+qtI/C9Co/fdz9adfj=
</data>`

(Für diesen Eintrag hier habe ich das examKeySalt inhaltlich, aber nicht strukturell verändert)

Tatsächlich funktioniert die Konfiguration ohne diesen Abschnitt dann zunächst problemlos. Bisweilen ist das examKeySalt für uns noch nicht relevant, insofern bleibt abzuwarten wie es sich künftig entwickelt.
